guidelines/TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+# Title, copied verbatim from the main overview page
+
+> Summary, copied verbatim from the main overview page
+
+## What?
+
+More detail on **what** the requirement is. Do not put anything to do with why the requirement exists here, that should be below in "Why?".
+
+### Requirements
+
+* List of requirements to meet the standard. Do not use platform specific language here where possible, that should be below in "How"?
+
+### WCAG Success Criteria
+
+* [\*.\*.\* WCAG Success Critera Link (A-Level)]()
+* [1.1.1 Non-text content (A)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)
```

Comment:
Left this in as an example, not sure if it's clear enough how the format should be if it's taken out